arkane/statmech.py

```diff
@@ -234,7 +234,7 @@ def load(self, pdep=False, plot=False):
         path = self.path
         directory = os.path.abspath(os.path.dirname(path))
 
-        def create_log(log_path):
+        def create_log(log_path, check_for_errors=True):
```

Comment:
Can you document this functionality to the Arkane documentation? I think the best place is at the end of this [section](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/documentation/source/users/arkane/input.rst#option-1-automatically-parse-quantum-chemistry-calculation-output) and just above this [header](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/documentation/source/users/arkane/input.rst#without-hinderedfree-rotors)

When it mentions the Log class, mention that the first argument is the path to the log file itself. Optionally, you may supply the keyword argument `check_for_errors=False` to prevent Arkane from raising an exception if it finds an error in the QM job based on the statements in the Log.

Reply:
Absolutely. Hopefully this is the last time I forget to update the corresponding documentation 😅 Let me know if I should modify the wording :)